src/components/WorkPoolStatusIcon.vue

```diff
@@ -0,0 +1,66 @@
+<template v-if="status">
```

Comment:
1. Is this v-if is needed?
2. Does this work on the root template? If so TIL

Reply:
Yep, the `v-if` is needed because push work pools will not display a status because the user won't need to host any workers. Push work pools will get a status of `null` from the API.

That `v-if` didn't work on the root template, so I moved it to the `p-tooltip`. Thanks for the catch!

Reply:
aw man, I was hoping it worked on that root template tag!